tests/models/test_model.py

```diff
@@ -516,12 +530,29 @@ def test_opt_wrapping():
 @pytest.mark.parametrize('tie_word_embeddings', [True, False])
 @pytest.mark.parametrize('expansion_ratio,ffn_hidden_size', [
     (2, None),
-    (1.231, None),
+    pytest.param(1.231,
+                 None,
+                 marks=pytest.mark.xfail(
+                     reason='d_model * expansion_ratio must be an integer.',
+                     strict=True)),
     (2, 128),
     (2, 256),
 ])
+@pytest.mark.parametrize('ffn_act_fn', [
+    {
+        'name': 'gelu',
+        'approximate': 'none'
+    },
+    {
+        'name': 'silu'
+    },
```

Comment:
```suggestion
    {'name': 'gelu', 'approximate': 'none'},
    {'name': 'silu'},
```
same as above

Reply:
pre-commit `yapf` undoes this...

not sure how to make it clean AND make `yapf` happy